content/logs/log_collection/_index.md

```diff
@@ -237,19 +237,21 @@ logs:
 
 ### Using a Proxy for Logs
 
-The log Agent does not presently respect the usual proxy setting in the `datadog.yaml` configuration file. It is **not yet** possible to use the agent as proxy (local agent -> proxy agent -> Datadog) but a third party proxy can be used.
+The log Agent does not presently respect the usual proxy setting in the `datadog.yaml` configuration file, since logs are only forwarded through TCP/SSL and not HTTPs.
 
-To use a third party proxy, configure your Datadog Agents to forward logs to this proxy by setting those parameters in their respective `datadog.yaml` configuration file:
+Configure your Datadog Agent to forward logs to a proxy by setting those parameters in your Agent  `datadog.yaml` configuration file:
```

Comment:
s/those/these/

Reply:
"To configure your Datadog Agent to forward logs through a proxy server, add these settings to the `datadog.yaml` configuration file:"